packaging/installer/methods/source.md

```diff
@@ -47,6 +54,19 @@ which the the build system will link statically into Netdata. These
 libraries and their header files must be copied into specific locations
 in the source tree to be used.
 
+### If using a git checkout
```

Comment:
```suggestion
### Copy files using a git checkout
```

Reply:
This is supposed to be an addendum for the specific case of them using a git checkout, not instructions on how to do so.

IOW, this is supposed to be us saying ‘Hey, if you’re building from a git repository, you need to make sure this is taken care of otherwise things won’t work.’.